README.md

```diff
@@ -17,6 +17,7 @@ Sample
                 { headerText: "Price", rowText: 'price', isSortable: true }
             ],
             pageSize: 5,
+            cssTableClass: 'table table-striped table-hover',
```

Comment:
Thats good but can you please change it to 'yourTableClassName' instead of bootstrapClassName